.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build Debezium
+
+on: [push, pull_request]
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Pre Install Setup
+        run: |
+          sudo service mysql-5.6 stop || sudo service mysql stop
+          sudo /etc/init.d/postgresql stop
+      - name: Install and Setup Maven
+        run: |
+          wget https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.zip
+          unzip -qq apache-maven-3.6.3-bin.zip
+          export M2_HOME=$PWD/apache-maven-3.6.3
+          export PATH=$M2_HOME/bin:$PATH
+          mvn -version
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+#  web-hooks:
```

Comment:
I think you can remove webhooks in all workflows. There is already and integration between Github and Gitter that sends PR notifications to the dev channel. The webhooks in Travis config file were used to send notifications about build results. AFAIK there is no such link between Githuba Actions and Gitter yet.

The list of available integrations is available at https://gitlab.com/gitlab-org/gitter/services/-/tree/master/lib

So a viable solution would be sending a message in for example Jenkins API format to Gitter's Jenkins integration endpoint.